src/main/java/org/kairosdb/core/datastore/KairosDatastore.java

```diff
@@ -248,21 +248,32 @@ public DatastoreQuery createQuery(QueryMetric metric) throws DatastoreException
 
 		DatastoreQuery dq;
 
-		try
+		Span span = tracer.buildSpan("create_query").start();
+
+		try(Scope scope = tracer.scopeManager().activate(span, false))
 		{
 			dq = new DatastoreQueryImpl(metric);
+			span.setTag("query_waiting_count", m_queuingManager.getQueryWaitingCount());
```

Comment:
Is creating this query a long-running enough of a process to merit an entirely new span, or should we just add this tag onto an existing parent span?